src/Libraries/Microsoft.Extensions.AI.Evaluation.Safety/Microsoft.Extensions.AI.Evaluation.Safety.json

```diff
@@ -0,0 +1,329 @@
+{
+  "Name": "Microsoft.Extensions.AI.Evaluation.Safety, Version=10.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35",
```

Comment:
The Safety package is still in preview and should not have the API json turned on. Could you please revert this file?